avm/ptn/app/container-job-toolkit/main.bicep

```
@@ -0,0 +1,341 @@
+metadata name = 'container-job-toolkit'
```

This will be the readme title, hence it usually is a short user friendly name, e.g., 
```suggestion
metadata name = 'Container job toolkit'
```